GenerateToolingFeed/V4Format/V4FormatFeedEntryUpdater.cs

```diff
@@ -6,17 +6,30 @@ namespace GenerateToolingFeed.V4Format
 {
     internal class V4FormatFeedEntryUpdater : IFeedEntryUpdater
     {
-        private static readonly IDictionary<string, string> _dotnetToTemplatesPrefix = new Dictionary<string, string>()
+        private static readonly IDictionary<string, string[]> _dotnetToItemTemplatesPrefix = new Dictionary<string, string[]>()
```

Comment:
This changed from a prefix to a fully qualified name. Preserving the name was causing some confusion